modules/features2d/misc/java/test/BRIEFDescriptorExtractorTest.java

```diff
@@ -28,7 +27,7 @@ private Mat getTestImg() {
     @Override
     protected void setUp() throws Exception {
         super.setUp();
-        extractor = DescriptorExtractor.create(DescriptorExtractor.BRIEF);
+        extractor = createAlgorythmInstance(XFEATURES2D_PACKAGE_NAME+"BriefDescriptorExtractor", ALGORITHM_FACTORY_NAME, null, null);
```

Comment:
Sounds good.
BTW, Why is this functionality limited for tests only? I believe it should be available for regular Java applications too.